cdc/sink/ddlsink/mysql/mysql_ddl_sink.go

```diff
@@ -62,6 +61,8 @@ type DDLSink struct {
 	// statistics is the statistics of this sink.
 	// We use it to record the DDL count.
 	statistics *metrics.Statistics
+	// map: model.TableName -> struct{}
+	runningAsyncDDLs sync.Map
```

Comment:
@3AceShowHand  @asddongmen @wk989898 PTAL, A cache is added here to quickly determine if a asyncCheck is needed.